docs/developing-component.md

```diff
@@ -45,6 +46,91 @@ git clone https://github.com/dapr/components-contrib.git github.com/dapr/compone
 make test
 ```
 
+### Debug unit-tests
+
+Specify the package that you want to test when running the `dlv test`. For example, to debug the `./pkg/actors` tests:
+
+```bash
+dlv test -v ./state/redis
+```
+
+If you want to combine VS Code + dlv for debugging so you can set breakpoints in the IDE create a debug launch configuration as follows:
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+    {
+        "name": "Launch test function",
+        "type": "go",
+        "request": "launch",
+        "mode": "test",
+        "program": "${workspaceFolder}/state/redis",
+        "buildFlags": "-v"
+    },
+]
+}
+```
+
+### Enabling conformance tests for your new component
+
+You will need to update [`tests/conformance/common.go`](https://github.com/dapr/components-contrib/blob/master/tests/conformance/common.go) and `tests/config/<component-type>/tests.yaml` ([bindings components example](https://github.com/dapr/components-contrib/blob/master/tests/config/bindings/tests.yml)) to add your component.
+
+Once conformance tests pass locally also add the test to the GitHub workflow at https://github.com/dapr/components-contrib/blob/master/.github/workflows/conformance.yml
```

Comment:
use [config](link) instead

Reply:
Not sure what this means, but it's no longer applicable to this PR.